MoltenVK/MoltenVK/GPUObjects/MVKPixelFormats.mm

```diff
@@ -792,6 +801,11 @@
 	return _mtlVertexFormatDescriptions[fmtIdx];
 }
 
+VkFormatFeatureFlags MVKPixelFormats::convertFormatPropertiesFlagBits(VkFormatFeatureFlags2 flags) {
+    // Truncate to 32-bits and just return. All current values are identical.
+    return static_cast<VkFormatFeatureFlags>(flags);
+}
```

Comment:
I like that you've put this in a separate function, rather than doing several static casts inline where it's used, in case we ever need to modify this, or add warning logging, etc.

Reply:
I initially misread the spec thinking that the values are not 1:1 the same for some reason, which made me put this into a function. After reading it more exactly I noticed that wasn't the case but decided to leave it. But this isn't anything to change, so I'll resolve.